netsim/extra/firewall.zonebased/plugin.py

```diff
@@ -0,0 +1,43 @@
+# Firewall Zone-Based Plugin
+## Simple rules for now, only default policy supported initially
+
+from netsim.utils import log
+from netsim.augment import addressing, devices
+from netsim import api,data
+from box import Box
+import netaddr
+
+_config_name = 'firewall.zonebased'
+
+def post_transform(topology: Box) -> None:
+    global _config_name
+    for node in topology.nodes.values():
+        features = devices.get_device_features(node,topology.defaults)
+        zbf_supported = 'firewall.zonebased' in features
+        if zbf_supported:
```

Comment:
This would create "firewall" and "firewall.zonebased" on every device that has this feature. Does that make sense? Wouldn't it be better to rely on lab topology to define these data structures and leave them missing if they are not defined?